### ISSUE TITLE
Blurry/Transparent title bar

### ISSUE THREAD

**Issue by gallowstree** [andreyorst/Breezemite#12]
I was using Breezemite with no problems for a couple of months. Now I reinstalled Kubuntu and the only thing that broke was Breezemite :( 

I love how it looked before, but now the title bar instead of having the nice light color is transparent with a blur "effect" but it looks really bad. See the screenshot, the title bar takes the color of whatever is behind it.

Could you please help me? I love this theme and it sucks not being able to use it.
![screenshot_20170924_213238](https://user-images.githubusercontent.com/6878435/30792165-759a6f6c-a175-11e7-871e-64dc72bffa54.png)

**Comment by andreyorst** [comment]
Whoa, Kubuntu uses such an old version of Plasma.. 

First of all - tell me specs of your display: resolution, DPI, etc.
I believe you're using a HiDPI display, aand thats a root of the issue.

Next - check if interface scaling is enabled or disabled. tip: https://github.com/andreyorst/Breezemite/issues/8#issuecomment-308749763

Last but not least - try the fix from here: https://github.com/andreyorst/Breezemite/issues/4#issuecomment-295890785

Basically you need to scale your theme manually in `breezemiterc` file. I think it'll do the trick

**Comment by gallowstree** [comment]
Thank you a lot, it looks just like #4 I think it will fix it too. Will try later today and let you know if it worked :) cheers and thanks for sharing your theme with the world.

I do have a HiDPI display, but it was not a problem with ubuntu + KDE. Something is different with the pure kubuntu that made it like this.

**Comment by gallowstree** [comment]
Doubling every value that represented a size or an offset did the trick, thanks a lot.
